### DISCUSSION TITLE
Sorting by "next step" and make a calender out of "next step"

### DISCUSSION THREAD

**Discussion by SneakyPie13** [community/community#63219] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi,

I have actually raised the issue before. We have since today the latest version, but unfortunately it was not yet implemented. It would be really great if you could sort by "next step" in the experiment overview. Then you would have directly a timeline after which you can follow.

I would also like the idea if you could add the next steps directly as a calendar entry. That would save a lot of work and would be really clear.

Thanks!

Ariane

**Comment by rainbow1016** [comment]
very good question!
